File: src/main/java/com/wgzhao/addax/admin/dto/TableMetaDto.java
package com.wgzhao.addax.admin.dto;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Schema(name = "TableMetaDto", description = "数据库表元信息（仅名称与注释）")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class TableMetaDto {
    @Schema(description = "表名")
    private String name;

    @Schema(description = "表注释")
    private String comment;
}

File: src/main/kotlin/com/wgzhao/addax/admin/dto/TableMetaDto.kt
package com.wgzhao.addax.admin.dto

import io.swagger.v3.oas.annotations.media.Schema

@Schema(name = "TableMetaDto", description = "数据库表元信息（仅名称与注释）")
data class TableMetaDto(
    @Schema(description = "表名")
    val name: String,

    @Schema(description = "表注释")
    val comment: String? = null
)
